docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -76,15 +80,36 @@ $ tctl create -f access.yaml
 
 #### Export access-plugin Certificate
 
-Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
-
-```bash
-$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=720h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuring-teleport-slack).
+The slack plugin uses the `access-plugin` role and user to perform the approval. Export credentials for the slack plugin to connect to Teleport using [`tctl auth sign`](https://goteleport.com/docs/cli-docs/#tctl-auth-sign).
```

Comment:
Slack plugin no longer performs the approval, let's update the role and the text